### ISSUE TITLE
Create a CI pipeline

### ISSUE THREAD

**Issue by rahearn** [usagov/pangolin#33]
The full eventual pipeline will require:

* unit tests
* a11y scan
* dynamic security scan
* static security scan
* dependency audits
* deployment to a staging environment
* deployment to production (maybe not until in a fully-moderate environment)

Break each piece into its own task to implement

**Comment by rahearn** [comment]
Use Github Actions for this, at least at first.

**Comment by rahearn** [comment]
@hartsick thoughts on adding a ruby style enforcer? I'm a fan of https://github.com/testdouble/standard (which is basically rubocop with Opinions)

**Comment by hartsick** [comment]
@rahearn I'm not the biggest style enforcer fan (feel it's often more burden than worth on small teams, and consistency and readability can sometimes be a tradeoff) but don't feel strongly about it!

**Comment by rahearn** [comment]
> if you have a strong preference go for it

I'm fine with tabling it until we have an issue with bikeshedding
